db/migrations/20190625175330_create_players.rb

```diff
@@ -7,6 +7,8 @@
       column :token,
              'uuid',
              default: Hanami::Model::Sql.function(:uuid_generate_v4)
+      column :room, Integer
+      column :position, Integer
```

Comment:
We don't want to modify existing migrations, as they only get run once. Let's make a new one.

Reply:
@jgayfer closing for now and will make new PRs based off of our talk.